man/mysqlslap.1

```diff
@@ -119,15 +119,11 @@ mysqlslap \-\-concurrency=5 \-\-iterations=20 \e
 .RE
 .\}
 .PP
-Tell the program to load the create, insert, and query SQL statements from the specified files, where the
-create\&.sql
-file has multiple table creation statements delimited by
-\';\'
-and multiple insert statements delimited by
-\';\'\&. The
+Tell the program to load the create, insert, and query SQL statements from the specified files, where the create\&.sql
+file has multiple table creation statements delimited by ';' and multiple insert statements delimited by ';'\&. The
 \fB\-\-query\fR
-file will have multiple queries delimited by
-\';\'\&. Run all the load statements, then run all the queries in the query file with five clients (five times each):
+file will have multiple queries delimited by ';'\&.
+Run all the load statements, then run all the queries in the query file with five clients (five times each):
```

Comment:
This causes extra space:
`by ';'.  Run all the load statements` as it was: `by ';'. Run all the load statements`

Reply:
Right. Re-flowed paragraph now.